ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/RaoUtil.java

```diff
@@ -160,4 +168,52 @@ public static ObjectiveFunctionEvaluator createObjectiveFunction(RaoParameters r
                 throw new NotImplementedException("Not implemented objective function");
         }
     }
+
+    public static List<Optional<Country>> getCnecLocation(BranchCnec cnec, Network network) {
```

Comment:
I'm wondering if it couldn't be interesting to have these `getLocation()` method into the Crac API instead of in a RAO utility class ?

In NetworkElement, Cnec, and NetworkAction classes.

To cross-check with other developers of the team. I don't have a strong opinion on what is best :p